### PR TITLE
check-dfsg-status: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.33

### CODE REVIEW

**Comment by Rudloff** [comment]
> The page pages/linux/check-dfsg-status.md seems to be a copy of pages/linux/vrms.md (52% matching).

Indeed, check-dfsg-status is the new name of vrms.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
> Indeed, check-dfsg-status is the new name of vrms.

We probably need to update the existing page to reflect the name change too.

File: pages/linux/check-dfsg-status.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# check-dfsg-status
+
+> Report non-free packages installed on Debian-based OSes.
```
Comment (fazlearefin) [inline pages/linux/check-dfsg-status.md:3]: Worth saying what this command used to be called previously.

```suggestion
> Report non-free packages installed on Debian-based OSes.
> This command was formerly known as `vrms`.
```

**Review by fazlearefin — APPROVED** [review]

**Pull request merged**